backend/src/main/java/io/debezium/configserver/rest/LifecycleResource.java

```diff
@@ -0,0 +1,278 @@
+            String errorMessage = "Could not connect to Kafka Connect! Kafka Connect REST API is not available at \""
+                    + kafkaConnectURI + "\". (" + e.getMessage() + ")";
+            LOGGER.error(errorMessage);
+            return Response.status(Status.SERVICE_UNAVAILABLE).entity(new ServerError(errorMessage, StacktraceHelper.traceAsString(e))).build();
+        }
+        LOGGER.error("Kafka Connect response: " + result);
+
+        return result;
+    }
+
+    @Path(CONNECTOR_RESUME_ENDPOINT)
+    @PUT
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @APIResponse(
+            responseCode = "200",
+            content = @Content(
+                    mediaType = MediaType.APPLICATION_JSON,
+                    schema = @Schema(implementation = ConnectionValidationResult.class)
+            ))
+    @APIResponse(
+            responseCode = "404",
+            description = "Invalid connector name provided",
+            content = @Content(
+                    mediaType = MediaType.APPLICATION_JSON,
+                    schema = @Schema(implementation = NotFoundResponse.class)
+            ))
+    @APIResponse(
+            responseCode = "500",
+            description = "Exception during action",
+            content = @Content(
+                    mediaType = MediaType.APPLICATION_JSON,
+                    schema = @Schema(implementation = ServerError.class)
+            ))
+    @APIResponse(
+            responseCode = "503",
+            description = "Exception while trying to connect to the selected Kafka Connect cluster",
+            content = @Content(
+                    mediaType = MediaType.APPLICATION_JSON,
+                    schema = @Schema(implementation = ServerError.class)
+            ))
+    public Response resumeConnector(
+            @PathParam("cluster") int cluster,
+            @PathParam("connectorname") String connectorName
+    ) {
+        KafkaConnectClient kafkaConnectClient;
+        URI kafkaConnectURI;
+        try {
+            kafkaConnectURI = KafkaConnectClientFactory.getKafkaConnectURIforCluster(cluster);
+            kafkaConnectClient = KafkaConnectClientFactory.getClient(cluster);
+
+        } catch (KafkaConnectException e) {
+            return Response.status(Status.INTERNAL_SERVER_ERROR).entity(e.getServerError()).build();
+        }
+
+        Response result;
+        try {
+            result = kafkaConnectClient.resumeConnector(connectorName);
+        } catch (ProcessingException | IOException e) {
+            String errorMessage = "Could not connect to Kafka Connect! Kafka Connect REST API is not available at \""
+                    + kafkaConnectURI + "\". (" + e.getMessage() + ")";
+            LOGGER.error(errorMessage);
+            return Response.status(Status.SERVICE_UNAVAILABLE).entity(new ServerError(errorMessage, StacktraceHelper.traceAsString(e))).build();
+        }
```

Comment:
As this seems to appear in multiple places, I'd suggest to extract an [exception mapper](https://docs.oracle.com/javaee/7/api/javax/ws/rs/ext/ExceptionMapper.html).

Reply:
Done